shared/actions/teams/index.js

```diff
@@ -283,16 +283,42 @@ const _createNewTeamFromConversation = function*(
   }
 }
 
+const isYouAreNotMemberError = (e: Error) => {
+  // TODO: Can we use a constant here instead?
+  const youAreNotMemberErrorCode = 2711
```

Comment:
This comes from the server, so I don't think there's an existing constant for it.

Reply:
can put it into the avdl or at least put it into constants/teams